test/common/testdata/audit/audit-configmap.yaml

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
I would rather add https://github.com/gardener/gardener-extension-registry-cache/blob/757bb0231c3b5539eb766b71871717b97ecd532a/.reuse/dep5#L31 to the `.reuse/dep5` file.

Reply:
Then I would do this for the entire examples folder. Other files here also contain the licensing header.